scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/Handshaker.java

```diff
@@ -563,26 +564,35 @@ protected void calculateKeys(byte[] masterSecret) {
 	 * concatenate a uint16 with the value N, N zero octets, a second uint16
 	 * with the value N, and the PSK itself.
 	 * 
-	 * @param psk
-	 *            the preshared key as byte array.
-	 * @return the premaster secret.
+	 * @param psk - preshared key as byte array
+	 * @param otherSecret - "other secret" either is zeros or comes from DHE_PSK.
```

Comment:
please update the method's description (JavaDoc) to reflect the role of the *other secret* and provide a pointer/link to the corresponding RFC